### DISCUSSION TITLE
一开始下载就卡死动不了且网络调用为0

### DISCUSSION THREAD

**Discussion by changtongwa** [ywmoyue/biliuwp-lite#288] [Q&A] [answered]
一开始下载就卡死动不了且网络调用为0

**Comment by ywmoyue** [comment, marked as answer]
1. windows是什么版本
2. 安装应用时是否使用脚本安装，如果不是请尝试卸载应用再使用脚本安装，然后再试试下载
3. 第二点不行后请将日志级别在应用设置其他页设置为Trace，并重试一遍下载，然后将日志文件通过邮件发送给我，日志文件在应用设置帮助页最底下能找到，我的邮箱地址在应用设置关于页能找到

**Reply by changtongwa** [reply]
<img width="711" alt="image" src="https://github.com/ywmoyue/biliuwp-lite/assets/136437951/e3d99409-a5ee-4120-9847-232ca155bc3c">
为什么用Y修改不了

**Reply by changtongwa** [reply]
已解决，谢谢啦
